co2eq/purchase/footprints.yml

```
@@ -1337,7 +1337,7 @@ _children:
             unit: kg
             source: 'https://www.sciencedirect.com/science/article/pii/S0959652618332633'
             conversions:
-              scoop:
+              portion:
                 kilograms: 0.068
```

since we now "loose" the label of what type of conversion this is, should we maybe add an `label` tag? Because if we want to display such kind of information to the user (which I think is important, as a "unit" is not very descriptive), it needs to get stored at some place.... 
```suggestion
                label: scoop
                kilograms: 0.068
```

---

You're right. That information should be stored in a formatting function though, where we keep at a central place the mapping between units and their label. I don't think it should be here as that would allow having different labels for the same units.